src/gradata/enhancements/behavioral_extractor.py

```
@@ -0,0 +1,478 @@
+
+
+def _try_llm_extract(llm_provider, draft: str, final: str, classification) -> str | None:
+    """Try LLM extraction, return result or None on failure."""
+    if llm_provider is None:
+        return None
+    try:
+        refined = llm_provider.extract(draft, final, classification)
+        if refined and _is_actionable(refined):
+            return refined
+    except Exception:
+        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)